algos/index.rst

```diff
@@ -13,7 +13,10 @@ build pipelines.
 
    "Asynchronous sample rate conversion", "Converts between common sample rates and connects pipelines with different clock domains", "Yes", "Xtensa HiFi3", "Upstream"
    "Channel selector", "Copies the selected channel from the source buffer to the sink buffer", "Yes", "N/A", "Upstream"
+   "Crossover", "Splits audio up into at most 4 different bands for individual processing", "Yes", "No", "Planned"
+   "DCBlocker", "Simpled highpass filter to remove DC components from audio", "Yes", "N/A", "Planned"
    "Demultiplexer", "Copies PCM sample frames from one source buffer to multiple sink buffers with configurable channels", "Yes", "N/A", "Upstream"
+   "Dynamic Range Processor", "Compresses audio range by bringing out quiet sounds and dampening loud sounds", "Yes", "Yes", "Planned"
```

Comment:
Nitpick but I think it has capability for both compress and expand, so maybe "Compress or expand audio signal to bring out quiet..."?

Reply:
That is correct, it has been many years since I tuned a DRC for a concert. Will update accordingly.

Reply:
@cujomalainey I will hold off on merging this pull request until you have made this update.